composer.json

```diff
@@ -18,11 +18,11 @@
   ],
   "require": {
     "php": ">=7.2.0",
-    "nette/caching": "~3.0.0",
+    "nette/caching": "~3.0",
     "psr/cache": "^1.0.0"
   },
   "require-dev": {
```

Comment:
^3.0.0

Reply:
I used the same constraint operator (tilde), as tilde had been used in the original composer.json. Semantically ~3.0 and ^3.0.0 are the same. So would you like me to update pull request to contain ^3.0.0 instead of ~3.0.0?

Reply:
Yep, I know it's the same. I want to be consistent and we are using ^3.0.0 at this moment.